README.md

```diff
@@ -3,16 +3,15 @@
 [![](http://meritbadge.herokuapp.com/dlib)](https://crates.io/crates/dlib)
 [![Docs.rs](https://docs.rs/dlib/badge.svg)](https://docs.rs/dlib)
 
-dlib is a small crate providing macros to make easy the use of external system libraries
-that can or cannot be optionally loaded at runtime, depending on whether the `dlopen` cargo
-feature is enabled.
+dlib is a small crate providing macros to make easy the use of external system libraries that
+can or cannot be optionally loaded at runtime, depending on whether a certain feature is enabled.
 
 ## Usage
 
-dlib defines the `external_library!` macro, which can be invoked with way:
+dlib defines the `external_library!` macro, which can be invoked in this way:
 
 ```rust
-external_library!(Foo, "foo",
+external_library!("dlopen-foo", Foo, "foo",
```

Comment:
I'm wondering if adding some macro syntax like this could be better, to make extra clear what is happening to people reading the code:

```
external_library!(feature="dlopen-foo", Foo, "foo");
```

And the same for other macros. What do you think about that?

Reply:
That's probably more obvious, also it allows a default feature name of "dlopen" so existing code that doesn't specify a feature name will still work if a "dlopen" feature is added to the consuming crate.